conversion/conversion.go

```diff
@@ -182,13 +186,36 @@ func GetMYSQLConnectionStr(server, port, user, password, dbname string) string {
 	return fmt.Sprintf("%s:%s@tcp(%s:%s)/%s", user, password, server, port, dbname)
 }
 
+func generateSQLSERVERConnectionStr() (string, error) {
+	server := os.Getenv("SSHOST")
+	port := os.Getenv("SSPORT")
+	user := os.Getenv("SSUSER")
+	dbname := os.Getenv("SSDATABASE")
```

Comment:
These and some other bits retain the older funcitonality of env variables. @bharadwaj-aditya when do we stop adding supporting for these?

Reply:
These unused env variables are now removed.

Reply:
I'm looking at the latest commit at it looks like we still retain the support for env variables.

Reply:
Yes, I may have added that comment early. These should be gone by now.